nutanix/data_source_nutanix_floating_ips.go

```diff
@@ -0,0 +1,217 @@
+package nutanix
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	v3 "github.com/terraform-providers/terraform-provider-nutanix/client/v3"
+)
+
+func dataSourceNutanixFloatingIPs() *schema.Resource {
```

Comment:
By file name it looks like a plural datasource but from implementation it looks like a singular datasource.